prov/efa/src/rxr/rxr_pkt_type_req.h

```diff
@@ -67,6 +67,7 @@
 #define RXR_REQ_TAGGED			BIT_ULL(3)
 #define RXR_REQ_RMA			BIT_ULL(4)
 #define RXR_REQ_ATOMIC			BIT_ULL(5)
+#define RXR_DATA_DELIVERY_COMPLETE		BIT_ULL(6)
```

Comment:
Adding a flag is a wire protocol change. Any wire protocol change need additional review and I prefer not to do that here. Can we check the ep's tx_op_flags instead?

Reply:
In `rxr_pkt_handle_data_send_completion`, we need to figure out that if we are using emulated read protocol or not.

Reply:
But I think that I can use an integer here, since the flag field of the data packet headers are all set to 0 currently https://github.com/ofiwg/libfabric/blob/master/prov/efa/src/rxr/rxr_pkt_cmd.c#L72

Reply:
Get rid from wire protocol change now, resolving.